module-system/sov-state/src/prover_storage.rs

```diff
@@ -188,6 +194,7 @@ impl<S: MerkleProofSpec> Storage for ProverStorage<S> {
             .expect("native db write must succeed");
 
         self.db.inc_next_version();
+        self.native_db.inc_next_version();
```

Comment:
Here line `196 & 197` looks like version should be shared and incremented only once?